minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -53,22 +69,25 @@ public static void main(String[] args) throws Exception {
     CommonUtils.sleepMs(Constants.SECOND_MS);
   }
 
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   private BlockWorker mWorker = null;
   private long mWorkerCapacityBytes;
   private int mUserBlockSize;
   private int mQuotaUnitBytes;
   private String mTachyonHome;
-  private String mWorkerDataFolder;
   private Thread mWorkerThread = null;
   private String mLocalhostName = null;
   private LocalTachyonMaster mMaster;
   private TachyonConf mMasterConf;
   private TachyonConf mWorkerConf;
+  private TachyonConf mClientConf;
+  private TachyonConf mTestConf;
```

Comment:
I thought `mTestConf` is more informative than `mTachyonConf`? It is kinda obvious this `conf` is a Tachyon Configuration, and I thought `mTestConf` makes the name aligned with `m{Master,Worker,Client}Conf`